packages/llm-mapper/mappers/openai/realtime.ts

```diff
@@ -147,59 +148,69 @@ export const mapRealtimeRequest: MapperFn<any, any> = ({
   };
 };
 
+
+const combineAudioBuffers = (audioBuffer: SocketMessage[], audio_key: "audio" | "delta", type: RealtimeMessage["type"]) => {
+  // When we hit a commit, we create a combined message from all appends in the current group
+  const firstMsg = audioBuffer[0];
+  const lastMsg = audioBuffer[audioBuffer.length - 1];
+
+  // Properly combine audio buffers - this is safer than just joining strings
+  // First, collect all base64 audio chunks
+  const audioChunks = audioBuffer
+    .map((m) => m.content[audio_key] || "")
+    .filter((chunk) => chunk.length > 0);
+
+  // If we have valid chunks, combine them
+  let combinedAudio = "";
+  if (audioChunks.length > 0) {
+    combinedAudio = audioChunks.join("");
+  }
```

Comment:
nit this the same thing as this
```suggestion
  let combinedAudio = audioChunks.join("");
```